tests/async_client_tests.rs

```diff
@@ -58,3 +58,38 @@ fn sync() {
 
     assert!(rt.block_on(client.sync_token()).is_some());
 }
+
+#[test]
```

Comment:
You can now use `#[tokio:test]` and test async functions dirrectly, no need to create runtimes manually, example: https://github.com/matrix-org/matrix-rust-sdk/blob/master/src/crypto/machine.rs#L853